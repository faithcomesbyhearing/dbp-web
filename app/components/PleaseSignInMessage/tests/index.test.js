import React from 'react';
import { render } from '@testing-library/react';
import { PleaseSignInMessage } from '..';

jest.mock('react-intl', () => ({
	FormattedMessage: ({ defaultMessage }) => <span>{defaultMessage}</span>,
	defineMessages: (messages) => messages,
}));

const props = {
	dispatch: jest.fn(),
	message: 'accessNotebook',
};

describe('<PleaseSignInMessage /> Component', () => {
	it('Should match the previous snapshot', () => {
		const { asFragment } = render(<PleaseSignInMessage {...props} />);
		expect(asFragment()).toMatchSnapshot();
	});
});
