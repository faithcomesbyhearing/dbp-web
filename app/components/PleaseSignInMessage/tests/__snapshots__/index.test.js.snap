// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PleaseSignInMessage /> Component Should match the previous snapshot 1`] = `
<DocumentFragment>
  <div
    class="need-to-login"
  >
    <span>
      Please
    </span>
     
    <span
      class="login-text"
      role="button"
      tabindex="0"
    >
      <span>
        sign in
      </span>
    </span>
     
    <span>
      to access your notebook.
    </span>
  </div>
</DocumentFragment>
`;
